### ISSUE TITLE
Add support for `TEXT_DATETIME`

### ISSUE THREAD

**Issue by CGenie** [JuliaDatabases/SQLite.jl#289]
For my DB I'm getting:
```
┌ Warning: Unsupported SQLite declared type TEXT_DATETIME, falling back to String type
└ @ SQLite ~/.julia/packages/SQLite/aDggE/src/SQLite.jl:416
```
I guess `TEXT_DATETIME` should be supported by Julia though I can't find it officially mentioned in the documentation:
https://www.sqlite.org/datatype3.html

**Comment by quinnj** [comment]
Yeah, we're talking about removing the warning since we can't possible cover all possible types since sqlite allows you to put "anything" as the column type (ref #279 )
